src/twisted/conch/ssh/filetransfer.py

```diff
@@ -498,6 +498,10 @@ def connectionLost(self, reason):
         """
         Clean all opened files and directories.
         """
+
+        # On Python2, we're an oldstyle class, so we can't use super().
+        FileTransferBase.connectionLost(self, reason)
```

Comment:
What is the point of this?

Reply:
I think in general, if the superclass has an implementation, even if it's empty right now, it's good practice to call it, in case it becomes non-empty in the future. This isn't a performance-critical section of code or anything.

Reply:
ok. Thanks